client/v2/autocli/flag/address.go

```diff
@@ -3,29 +3,28 @@ package flag
 import (
 	"context"
 
-	reflectionv2alpha1 "cosmossdk.io/api/cosmos/base/reflection/v2alpha1"
 	"github.com/cosmos/cosmos-sdk/types"
 	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
 type addressStringType struct{}
 
 func (a addressStringType) NewValue(ctx context.Context, b *Builder) Value {
-	if b.AddressPrefix == "" {
-		conn, err := b.GetClientConn()
-		if err != nil {
-			panic(err)
-		}
-		reflectionClient := reflectionv2alpha1.NewReflectionServiceClient(conn)
-		resp, err := reflectionClient.GetConfigurationDescriptor(ctx, &reflectionv2alpha1.GetConfigurationDescriptorRequest{})
-		if err != nil {
-			panic(err)
-		}
-		if resp == nil || resp.Config == nil {
-			panic("bech32 account address prefix is not set")
-		}
-		b.AddressPrefix = resp.Config.Bech32AccountAddressPrefix
-	}
+	// if b.AddressPrefix == "" {
```

Comment:
This does not work when enriching the application directly.

Reply:
Fixed in https://github.com/cosmos/cosmos-sdk/pull/15906/commits/1d7664fb0387db2e6890b696cdd0fef3559f9f37 and blocked by https://github.com/cosmos/cosmos-sdk/pull/15913